istio.io/configmap-nginx.yaml

```diff
@@ -89,8 +89,11 @@ data:
           return 200 "ok\n";
         }
 
-        # Proxy to past versioned instances of the site.
-        location ~ ^/v-[0-9]+\.[0-9]+.*$ {
+        # Proxy version prefixed URL to versioned instances of
+        # https://istio.io/docs/ and corresponding css/js/image
+        location ~ ^/v-[0-9]+\.[0-9]+(/|/docs/?.*|/js/.*|/img/.*|/css/.*)?$ {
```

Comment:
`^/v-[0-9]+.[0-9]+(/docs|/js|/css)(/$)` is even better to deal with istio.io/v-0.1/docs too (yet not leaving un redirected something like /jscript/blah)